src/components/Content/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Wrapper from '../Wrapper';
+import ErrorBoundary from '../ErrorBoundary';
+import Filter from '../MovieFilter';
+import SearchResults from '../SearchResults';
+import './style.css';
+
+const Content = ({ searchResults }) => (
+  <main>
+    <ErrorBoundary>
+      <Wrapper>
+        <Filter/>
+        <SearchResults searchResults={searchResults}/>
+      </Wrapper>
+    </ErrorBoundary>
+  </main>
+);
+
+Content.propTypes = {
```

Comment:
Do not sure that "is required" is necessary for all properties as they can be absent from API.

Reply:
Thanks for your note. I will keep that in mind.